### PR TITLE
docs(gateways): add warning about wsresponse serialization

### PR DESCRIPTION
**Pull request by mukaschultze**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/nestjs/docs.nestjs.com/blob/master/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[x] Other... Please describe: Docs
```

## What is the current behavior?

Issue Number: nestjs/nest/issues/5525

## What is the new behavior?
`WsResponse` serialization is properly documented.

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

## Other information

### CODE REVIEW

**Comment by jmcdo29** [comment]
LGTM

**Comment by BrunoS3D** [comment]
LGTM

LGTM

**Review by BrunoS3D — APPROVED** [review]

**Review by jmcdo29 — APPROVED** [review]

File: content/websockets/gateways.md [hunk 1]
```diff
@@ -128,6 +128,8 @@ handleEvent(data) {
 
 > info **Hint** The `WsResponse` interface is imported from `@nestjs/websockets` package.
 
+> warning **Warning** You should return a class instance that `implements WsResponse` if your `data` field relies on Nest's serialization, as it ignores plain JavaScript objects responses.
```
Comment (kamilmysliwiec) [inline content/websockets/gateways.md:131]: We should explicitly mention the `ClassSerializerInterceptor` here as "Nest's serialization" seems to be somewhat ambigious

**Comment by kamilmysliwiec** [comment]
Thank you for your contribution! I left 1 minor comment :)

**Pull request merged**